website/docs/api/top-level.md

```diff
@@ -322,6 +321,7 @@ factories.
 | `readers`         | Registry for training and evaluation data readers like [`Corpus`](/api/corpus).                                                                                                                                                                    |
 | `schedules`       | Registry for functions that create [schedules](https://thinc.ai/docs/api-schedules).                                                                                                                                                               |
 | `tokenizers`      | Registry for tokenizer factories. Registered functions should return a callback that receives the `nlp` object and returns a [`Tokenizer`](/api/tokenizer) or a custom callable.                                                                   |
+| `misc`            | Registry for miscellaneous functions that return data assets, knowledge bases or anything else you may need.                                                                                                                                       |
```

Comment:
I alphabetized this list as it was getting too long to look up a registry quickly. Shall we keep it so and move this between `losses` and `optimizers`?

Reply:
Ahhh forgot this was alphabetized 😄 So yes, let's put it in the correct place then.